roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -72,6 +72,16 @@ def get_var(task_vars, *keys, **kwargs):
         raise OpenShiftCheckException("'{}' is undefined".format(".".join(map(str, keys))))
     return value
 
+def normalized_release(task_vars):
```

Comment:
@rhcarvalho is there a reason not to just make these (including get_var) static methods on OpenShiftCheck? Then we wouldn't need to import them explicitly...